packages/blitz/src/render.rs

```diff
@@ -788,6 +788,58 @@ impl ElementCx<'_> {
                                 }
                             }),
                         );
+
+                    if let Some(underline) = &style.underline {
+                        let mut draw_underline = |x: f32, y: f32, w: f32| {
```

Comment:
Perhaps this function (with a couple of extra params) should be shared between underline and strikethrough?